package.js

```diff
@@ -0,0 +1,20 @@
+Package.describe({
+  name: 'useful:collection-mocks',
+  version: '0.0.1',
+  summary: 'Adds a method to Mongo.Collection\'s prototype that mocks the results of a db operation',
+  git: 'https://github.com/usefulio/meteor-collection-mocks.git',
```

Comment:
Commit straight to master works.



—
Sent from Mailbox

On Wed, Aug 5, 2015 at 3:33 AM, Vlad Balan <notifications@github.com>
wrote:

>> @@ -0,0 +1,20 @@
>> +Package.describe({
>> +  name: 'useful:collection-mocks',
>> +  version: '0.0.1',
>> +  summary: 'Adds a method to Mongo.Collection\'s prototype that mocks the results of a db operation',
>> +  git: 'https://github.com/usefulio/meteor-collection-mocks.git',
> This may or may not be important, but what would be best practice for quick edits like this, fixing typos etc.? Should I commit straight to master, create a separate branch just for them or incorporate them into the next feature?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/usefulio/collection-mocks/pull/2/files#r36279746